app/controllers/sessions_controller.rb

```diff
@@ -11,11 +11,12 @@ def can_authenticate(user)
   def create
     @user = User.find_by(email: params.dig(:session, :email)&.downcase)
     if can_authenticate(@user)
```

Comment:
Shouldn't can_authenticate return a boolean value ? if so I think you need to change it to can_authenticate?(@user)

```suggestion
    if can_authenticate?(@user)
```

Reply:
yup, common convention in Ruby is to add a question mark at the end to signal that it returns a boolean :)